docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.4'
 
 x-elasticsearch-image: &elasticsearch-image
-  docker-registry.wikimedia.org/releng/cirrus-elasticsearch:7.10.2-s0
+  ghcr.io/wmde/wikibase/elasticsearch:7443724965
```

Comment:
This might be a problem. I would prefer not to update it.

Reply:
@eloiferrer what do you think?

Reply:
How can it be differently? I don't find a tag like :latest

Reply:
It took quite a while to find that the latest version of Cirrussearch extension did only work with that particular version. Note that if we change the ES container version, we will need to reindex everything. Maybe, we can do that in the follow-up?

Reply:
Ah ok, I misunderstood your comment. You are just referring to this particular image.
I thought you meant updating these tags in the future would be problematic. 
We can just keep the previous elasticsearch image.